src/Util/SupportedAddonsLoader.php

```diff
@@ -14,9 +14,12 @@ class SupportedAddonsLoader extends ApiLoader
      */
     public function getAddonNames()
     {
-        $endpoint = 'https://raw.githubusercontent.com/silverstripe/supported-modules/5/modules.json';
+        $endpoint = 'https://raw.githubusercontent.com/silverstripe/supported-modules/main/repositories.json';
```

Comment:
Look at changing `$this->doRequest()` to the new method in supported-modules

If non-trivial then don't bother

Reply:
This class is only used in one place, and the level of caching done by `ApiLoader` for this is unnecessary.
I'm deprecating this class and just updating `BringYourOwnIdeas\Maintenance\Tasks\UpdatePackageInfoTask::getSupportedPackages()` to use the PHP API instead.

I'll leave this implementation as it is now in case someone is calling this method for their own nefarious purposes.